tests/sentry/notifications/test_helpers.py

```diff
@@ -221,3 +223,32 @@ def test_get_team_members(self):
         assert get_team_members(team1) == [RpcActor.from_object(user1)]
         assert get_team_members(team2) == [RpcActor.from_object(user2)]
         assert get_team_members(team3) == []
+
+    def test_team_is_valid_recipient(self):
+        team1 = self.create_team(organization=self.organization)
+        team2 = self.create_team(organization=self.organization)
+        team3 = self.create_team(organization=self.organization)
+        integration1 = self.create_integration(
+            organization=self.organization, provider="Slack", external_id="slack-id"
+        )
+        integration2 = self.create_integration(
+            organization=self.organization, provider="Jira", external_id="jira-id"
+        )
+        ExternalActor.objects.create(
```

Comment:
should i use the replica in the test as well? i couldn't find any examples in the tests where the replica was used to create instances of an object instead of the original

Reply:
@corps thoughts?

Reply:
Replicas will be created automatically -- the outbox and replication logic runs on `save`.  In general, *do not write to the replicate yourself!*.  The replica models are intended to be controlled 100% by the replication logic.  It will mostly "just work" without intervention.  Check our test in `test_replica.py` -- the replicas are created implicitly from the factories.

Reply:
i guess my concern was more whether it was worth setting this test suite to run in control as well, in which case these lines would need to be changed to use the replica instead. or is it okay to leave this without the silo decorators since it's testing helper functions?

Reply:
oh, you can use `with assume_test_silo_mode(SiloMode.XXXX)` to make certain test code cross silo safe.